app/contacts/[id]/page.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+import Link from 'next/link';
+import Image from 'next/image';
+import { useParams } from 'next/navigation';
+import { useContacts } from '../../contexts/ContactsContext';
+
+export default function Contact() {
+  const { id } = useParams();
+  const { getContact } = useContacts();
+  const contact = getContact(id);
+
+  if (!contact) {
+    return (
+      <div>Contact not found</div>
+    );
+  }
+
+  return (

```

Comment:
every time you return something, the type itself needs to be declared in the function as well